src/App.tsx

```diff
@@ -32,7 +33,12 @@ export const App: React.FC<AppProps> = ({ args = [], options = {} }) => {
 
   return (
     <BrowserSessionProvider value={session}>
-      {session ? <UserControls /> : <Text>Loading...</Text>}
+      {session ? 
+        <>
+          <SongProgress />
+          <UserControls />
+        </> 
```

Comment:
This is fine for now but ultimately I think that we should consider some type of wrapper around all the components that will help us better control the layout and separate concerns.